ldk/javascript/src/whisper/index.ts

```diff
@@ -265,6 +284,146 @@ export interface WhisperAptitude {
   create(whisper: NewWhisper): Promise<Whisper>;
 }
 
+function convertChildComponent(component: BoxChildComponents): OliveHelps.ChildComponents {
```

Comment:
Will it be cleaner to move it to a mapper layer?

Reply:
My vote is a new file: `src/whisper/whisper-mapper.ts`. I envision a future where most aptitudes could end up with an associated mapper.

Reply:
Depends on what you mean by a mapper layer - we can move the conversion logic into a separate file, absolutely.

Reply:
I've moved the conversion logic into `convert.ts`. I've also moved the types into `types.ts` to avoid any cyclic dependency issues down the road.

Reply:
Yes, that is what I meant ;) Also, for consistency, could we call it map instead of convert? We already have mapper.ts and I think Cory is adding whisper-mapper.ts or something along this line. I think we should stick with 1 naming convention: convert or map.